Include/object.h

```diff
@@ -766,6 +766,7 @@ PyAPI_FUNC(void) _Py_NegativeRefcount(const char *filename, int lineno,
                                       PyObject *op);
 PyAPI_FUNC(void) _Py_INCREF_IncRefTotal(void);
 PyAPI_FUNC(void) _Py_DECREF_DecRefTotal(void);
+PyAPI_FUNC(void) _Py_IncRefTotal(PyInterpreterState *);
```

Comment:
I don't think we want to move this

Reply:
This is necessary because `_Py_TryIncref` ends up pulling it in in debug builds.